### ISSUE TITLE
It doesn't seem to correctly find aliases in tsconfig

### ISSUE THREAD

**Issue by fregante** [antoine-coulon/skott#20]
You can see my config at https://github.com/pixiebrix/pixiebrix-extension/blob/ae55fca9e0c610c801f05107f2cde021db09ef0f/tsconfig.json#L19-L26

I'm using a similar config in Madge and I think that one works: https://github.com/pixiebrix/pixiebrix-extension/blob/ae55fca9e0c610c801f05107f2cde021db09ef0f/package.json#L22
```sh
 npx skott src/pageScript/pageScript.ts --trackBuiltinDependencies --trackThirdPartyDependencies --displayMode=file-tree

 Running Skott from entrypoint: src/pageScript/pageScript.ts

 Processed 1 file (13.471ms)

 ✓ no circular dependencies found (depth=Infinity)

  pageScript.ts

 npm third-party dependencies: 

  @/contentScript (imports=1)
  @/frameworks (imports=1)
  @/messaging (imports=1)
  @/pageScript (imports=1)
  @/runtime (imports=1)
  @/types (imports=1)
  @/utils (imports=1)
  @/vendors (imports=1)
  lodash (imports=1)

 ✓ no builtin dependencies found

 Skott exited with code 0
```

**Comment by antoine-coulon** [comment]
Hello @fregante 

Thanks for reporting that issue!
🙂 

You're right, it seems that it does not resolve correctly your path aliases, I didn't cover the "@/" alias correctly, that's why it turns out that skott considers them in a fallback way as third party dependencies.

I'm going to fix that very quickly    so that you'll be able to use skott on the great PixieBrix extension 👌

Thanks again!

**Comment by antoine-coulon** [comment]
Hello @fregante,

Just published a **v0.12.3** of skott embedding a fix for this issue.

I tried `skott src/pageScript/pageScript.ts --trackBuiltinDependencies --trackThirdPartyDependencies --displayMode=file-tree` and it seems to resolve correctly path aliases now.

Feel free to re-open this issue or a new one if you ever notice the same or another problem, thanks to you we just made Skott better 😄

**Comment by fregante** [comment]
Thank you! Will try it soon
